frontend/src/components/connections/AlbyConnectionCard.tsx

```diff
@@ -86,40 +92,42 @@ function AlbyConnectionCard({ connection }: { connection?: App }) {
                     </LoadingButton>
                   </DialogTrigger>
                   <DialogContent>
-                    <DialogHeader>Link to Alby Account</DialogHeader>
-                    <DialogDescription className="flex flex-col gap-4">
-                      After you link your account, your lightning address and
-                      every app you access through your Alby Account will handle
-                      payments via the Hub.
-                      <img
-                        src="/images/illustrations/alby-account-dark.svg"
-                        className="w-full hidden dark:block"
-                      />
-                      <img
-                        src="/images/illustrations/alby-account-light.svg"
-                        className="w-full dark:hidden"
-                      />
-                      You can add a budget that will restrict how much can be
-                      spent from the Hub with your Alby Account.
-                    </DialogDescription>
-                    <div>
-                      <BudgetRenewalSelect
-                        value={budgetRenewal}
-                        onChange={setBudgetRenewal}
-                      />
-                      <BudgetAmountSelect
-                        value={maxAmount}
-                        onChange={setMaxAmount}
-                      />
-                    </div>
-                    <DialogFooter>
-                      <LoadingButton
-                        onClick={() => linkAccount(maxAmount, budgetRenewal)}
-                        loading={loading}
-                      >
-                        Link to Alby Account
-                      </LoadingButton>
-                    </DialogFooter>
+                    <form onSubmit={onSubmit}>
+                      <DialogHeader>Link to Alby Account</DialogHeader>
+                      <DialogDescription className="flex flex-col gap-4">
+                        After you link your account, your lightning address and
+                        every app you access through your Alby Account will
+                        handle payments via the Hub.
+                        <img
+                          src="/images/illustrations/alby-account-dark.svg"
+                          className="w-full hidden dark:block"
+                        />
+                        <img
+                          src="/images/illustrations/alby-account-light.svg"
+                          className="w-full dark:hidden"
+                        />
+                        You can add a budget that will restrict how much can be
+                        spent from the Hub with your Alby Account.
+                      </DialogDescription>
+                      <div>
```

Comment:
Can we also do this:
```suggestion
                      <div className="mt-4">
```



That would change:
<table>
<tr>
<td>
<img width="466" alt="Screenshot 2024-10-22 at 8 11 23 PM" src="https://github.com/user-attachments/assets/2064ccbd-4688-4d3f-8ed3-ebca8a355e67">
</td>

<td>
<img width="468" alt="Screenshot 2024-10-22 at 8 11 10 PM" src="https://github.com/user-attachments/assets/da877ba3-e21d-488c-8d79-74902b755769">
</td>
</tr>
</table>

Reply:
Thanks!